src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
```suggestion
    <title>Hello world</title>
```
Would be great to add some title

Reply:
Thanks for approving! Yes, I ran the `npm run deploy` command 2 times.
Also, should I make changes that you've suggested and push the code once again or these are the remarks to consider for future coding?